src/build-extensions-container.sh

```diff
@@ -1,20 +1,21 @@
 #!/bin/bash
-#Used by cmd/build-extensions-container.go
-#Find the RHCOS ociarchive.
 set -euo pipefail
```

Comment:
should we `set -x` here? since we remove the `set -x` from the code below?

Reply:
I've restored `set -x` for the key invocations!